app/components/agenda/agendaitem/create-agendaitem.hbs

```diff
@@ -24,77 +14,99 @@
+                      @checked={{includes subcase this.selectedSubcases}}
+                      {{on "input" (fn this.selectSubcase subcase)}}
+                    />
+                  </td>
+                  <td class="auk-table__col--11">
+                    {{subcase.shortTitle}}
+                    <br />
+                    <p
+                      data-test-create-agendaitem-row-subcase-name
+                      class="auk-u-text-muted"
+                    >
+                      {{capitalize subcase.subcaseName}}
+                    </p>
+                  </td>
+                </tr>
+              {{/each}}
+            </tbody>
+          </table>
+          <div class="auk-u-mx-4">
+            <Auk::Toolbar>
+              <Auk::Toolbar::Group @position="left">
+                <Auk::Toolbar::Item>
+                  <Auk::Pagination
+                    @page={{this.page}}
+                    @size={{this.size}}
+                    @nbOfItems={{this.subcases.length}}
+                    @total={{this.subcases.meta.count}}
+                    @onNextPage={{this.nextPage}}
+                    @onPreviousPage={{this.prevPage}}
+                  />
+                </Auk::Toolbar::Item>
+              </Auk::Toolbar::Group>
+              <Auk::Toolbar::Group @position="right">
+                <Auk::Toolbar::Item>
+                  {{t "amount-per-page"}}
+                </Auk::Toolbar::Item>
+                <Auk::Toolbar::Item>
+                  <WebComponents::AuPageSize
+                    @selectedSize={{this.size}}
+                    @onChange={{this.selectSize}}
                   />
-                </td>
-                <td class="auk-table__col--11">
-                  {{subcase.shortTitle}}
-                  <br>
-                  <p data-test-create-agendaitem-row-subcase-name class="auk-u-text-muted">
-                    {{capitalize subcase.subcaseName}}
-                  </p>
-                </td>
-              </c.body>
-            </table.content>
-          </DataTable>
+                </Auk::Toolbar::Item>
+              </Auk::Toolbar::Group>
+            </Auk::Toolbar>
+          </div>
         {{/if}}
       </div>
-    </Auk::Modal::Body>
-    <WebComponents::VlModalFooter
-      @isLoading={{this.isLoading}}
-      @disableSave={{this.noItemsSelected}}
-      @cancelButtonText={{t "cancel"}}
-      @saveButtonText={{t "add-to-agenda"}}
-      @cancelAction={{action "close"}}
-      @saveAction={{action "addSubcasesToAgenda"}}
-    />
-  {{/if}}
-</WebComponents::VlModal>
\ No newline at end of file
+    {{/if}}
+  </Auk::Modal::Body>
+  <Auk::Modal::Footer @onCancel={{@onClose}}>
+    <AuButton
+      data-test-create-agendaitem-save
+      @skin="primary"
+      @loading={{this.addSubcasesToAgenda.isRunning}}
+      @disabled={{this.isSaveDisabled}}
+      {{on "click" this.addSubcasesToAgenda.perform}}
```

Comment:
Use `(perform this.addSubcasesToAgenda)`

Reply:
will change